Benchmarks: add scripts for FastRNNs results comparison.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#44134 Benchmarks: add scripts for FastRNNs results comparison.**

Differential Revision: [D23505810](https://our.internmc.facebook.com/intern/diff/D23505810)

Commit:
Benchmarks: add scripts for FastRNNs results comparison.

[ghstack-poisoned]

bertmaher commented on benchmarks/compare-fastrnn-results.py:
```diff
@@ -0,0 +1,55 @@
+              'md'    : '| {:48s} | {:>13s} | {:>15s} | {:>10s} |',
+              'csv'   : '{:s}, {:s}, {:s}, {:s}'}
+data_fmt = {'table' : '{:48s} {:13.6f} {:15.6f} {:9.1f}%',
+            'md'    : '| {:48s} | {:13.6f} | {:15.6f} | {:9.1f}% |',
```
I love the table/md/csv formatting options <3

Commit:
Update on "Benchmarks: add scripts for FastRNNs results comparison."


Differential Revision: [D23505810](https://our.internmc.facebook.com/intern/diff/D23505810)

[ghstack-poisoned]
